src/examples/deploy_application_with_oidc.yml

```diff
@@ -0,0 +1,27 @@
+description: >
+  Deploy an application to AWS CodeDeploy with OIDC authentication
+usage:
+  version: 2.1
+  orbs:
+    aws-code-deploy: circleci/aws-code-deploy@x.y
```

Comment:
update `x.y` to show the proper version number for the next release based on this type of change. We can also update this in a later PR just before publishing.